saucelabs.conf.js

```diff
@@ -77,7 +78,7 @@ const setupConfig = {
     },
     autoDelay: {
       enabled: true,
-      delayAfter: 2000,
+      delayAfter: 1000,
```

Comment:
we have a helper named stepsListener might that work for you instead of this?

Reply:
the `stepListener.js` helper might be useful, but I think that might make sense to investigate in a separate ticket as the delays are far shorter (default 200ms) than what's been set here. So I'd want to run a fair number of test cycles, and knowing how fragile the crossbrwoser pipeline can be I wouldn't be surprised if it led to further tweaking of the timings.

Definitely worth investigating, but given the benefit the other improvements on this branch I reckon it might be best for a separate PR